### ISSUE TITLE
How to specify quotes in --probe?

### ISSUE THREAD

**Issue by lorien** [zmap/zgrab2#210]
Command
```
zgrab2 -f inp_httpbin banner -p 80 --probe 'POST /post HTTP/1.1\r\nHost: httpbin.org\r\nContent-Length: 5\r\n\r\na"b"c'
```

returns

```
panic: Probe error

goroutine 1 [running]:
github.com/zmap/zgrab2/modules/banner.(*Scanner).Init(0xc00037e060, 0xc96ee0, 0xc0002e8060, 0x6, 0xc00029c998)
        /root/go/src/github.com/zmap/zgrab2/modules/banner/scanner.go:103 +0x1ba
main.main()
        /root/go/src/github.com/zmap/zgrab2/cmd/zgrab2/main.go:127 +0xd18
```

What am I doing wrong? I need to send probe that contains quote character: `"`

**Comment by phspade** [comment]
mine is: FATA[0000] could not parse flags: unknown flag `probe'

**Comment by phspade** [comment]
unlike zgrab we can fetch a http request file with data flag. how do we do that in zgrab2?

**Comment by AnthraX1** [comment]
Use triple slash to escape. Should work with \\\"

**Comment by AnthraX1** [comment]
> unlike zgrab we can fetch a http request file with data flag. how do we do that in zgrab2?

I'll implement it after TLS

**Comment by zakird** [comment]
This is now resolved from #269.
